bubo/config.py

```diff
@@ -100,7 +100,7 @@ def __init__(self, filepath):
             if not re.match(re.compile(RoomRegex), admin) and not re.match(re.compile(UserIdRegex), admin):
                 raise ConfigError(f"Admin {admin} does not look like a user or room")
         self.coordinators = self._get_cfg(["permissions", "coordinators"], default=[])
-        for coordinator in self.admins:
+        for coordinator in self.coordinators: #FIXED ISSUE 16
```

Comment:
```suggestion
        for coordinator in self.coordinators:
```

Nice find! Copy paste mistake on my part.

No need to add the comment though, we have git log for changes.